temba/flows/views.py

```diff
@@ -2211,8 +2211,7 @@ def get_context_data(self, *args, **kwargs):
             if contact_query:
                 try:
                     org = flow.org
-                    group = org.all_groups.filter(group_type="A").first()
-                    contact_ids = query_contact_ids(org, contact_query, group=group)
+                    contact_ids = query_contact_ids(org, contact_query, active_only=False)
```

Comment:
@ihor-palii - Just to confirm: we don't need the group here anymore, right?

Reply:
@teehamaral - Yes, because that group was necessary to filter by active contacts.